1832.check-if-the-sentence-is-pangram.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * URL of this problem
+ * https://leetcode.com/problems/check-if-the-sentence-is-pangram/
+ */
+
+/**
+ * @param {string} sentence
+ * @return {boolean}
+ */
+var checkIfPangram = function(sentence) {
+    
+    const alphabet = "abcdefghijklmnopqrstuvwxyz";
+    
+    let counter = 0;
+    
+    if (sentence.length < 26) {
```

Comment:
where does 26 come from? Is it easy to understand for readers?

Reply:
I corrected this 26 to alphabet.length.